src/components/screens/wallet/transactions/transactionsOverview.js

```diff
@@ -56,8 +58,12 @@ class TransactionsOverview extends React.Component {
   }
 
   render() {
+    const { t, address, networkName } = this.props;
     const isSmallScreen = this.isSmallScreen();
     const filters = this.generateFilters();
+    // TODO remove the hardcode network once that the URL be available in production
+    // for mainnet and testnet. Right now ONLY works for testnet STAGIN mode.
+    const csvLink = `https://testnet-service-staging.lisk.io/api/v1/account/${address}/transactions/csv`;
```

Comment:
Once the API is ready, it should IMO somehow use a modification `getServerUrl` that doesn't throw an error
https://github.com/LiskHQ/lisk-hub/blob/390155c5c2a1383ff9f794503399beea3c4c0295/src/utils/api/lsk/liskService.js#L13-L22